tools/paths.py

```diff
@@ -76,12 +76,6 @@
 VODAFONE_LIBRARY = join(NET_LIBRARIES, "VodafoneUSBModem")
 UBLOX_LIBRARY = join(NET_LIBRARIES, "UbloxUSBModem")
 
-# FS
-FS_PATH = join(LIB_DIR, "fs")
-FAT_FS = join(FS_PATH, "fat")
-SD_FS = join(FS_PATH, "sd")
-FS_LIBRARY = join(BUILD_DIR, "fat")
```

Comment:
@sg- Just to confirm, fs library moves out of unsupported folder, gets all this updates, thus these 4 lines shall be removed and any dependencies on these (there are some mbed 2 tests that shall be removed then).